Documentation/observability/hubble-exporter.rst

```diff
@@ -151,3 +151,80 @@ The following is a complete example of configuring Hubble Exporter.
 
        {"flow":{"time":"2023-08-21T12:12:13.517394084Z","verdict":"DROPPED","IP":{"source":"fe80::64d8:8aff:fe72:fc14","destination":"ff02::2","ipVersion":"IPv6"},"l4":{"ICMPv6":{"type":133}},"source":{},"destination":{},"node_name":"kind-kind/kind-worker","drop_reason_desc":"INVALID_SOURCE_IP"},"node_name":"kind-kind/kind-worker","time":"2023-08-21T12:12:13.517394084Z"}
        {"flow":{"time":"2023-08-21T12:12:18.510175415Z","verdict":"DROPPED","IP":{"source":"10.244.1.60","destination":"10.244.1.5","ipVersion":"IPv4"},"l4":{"TCP":{"source_port":44916,"destination_port":80,"flags":{"SYN":true}}},"source":{"namespace":"default","pod_name":"xwing"},"destination":{"namespace":"default","pod_name":"deathstar-7848d6c4d5-th9v2"},"node_name":"kind-kind/kind-worker","drop_reason_desc":"POLICY_DENIED"},"node_name":"kind-kind/kind-worker","time":"2023-08-21T12:12:18.510175415Z"}
+
+
+Dynamic exporter configuration
+==============================
+
+Standard hubble exporter configuration accepts only one set of filters and
+requires cilium pod restart to change config. Dynamic flow logs allow configuring
+multiple filters at the same time and saving output in separate files.
+Additionally it does not require cilium pod restarts to apply changed configuration.
+
+**Dynamic Hubble Exporter** is enabled with Config Map property. It is disabled
+until you set a file path value for ``hubble-flowlogs-config-path``.
+
+.. include:: ../installation/k8s-install-download-release.rst
+
+Install cilium with dynamic exporter enabled:
+
+.. code-block:: shell-session
+
+   helm install cilium |CHART_RELEASE| \
+      --set hubble.enabled=true \
+      --set hubble.export.dynamic.enabled=true
+
+Wait for ``cilium`` pod to become ready:
+
+.. code-block:: shell-session
+
+    kubectl -n kube-system rollout status ds/cilium
+
+Verify that flow logs are stored in target files:
+
+.. code-block:: shell-session
+
+    kubectl -n kube-system exec ds/cilium -- tail -f /var/run/cilium/hubble/events.log
+
+You can edit configmap to change flow log settings without a need for pod to be
+restarted (changes should be reflected within 60s because of configmap
+propagation delay):
+
+.. code-block:: shell-session
+
+    kubectl -n kube-system edit cm cilium-flowlog-config
```

Comment:
This could be a `helm upgrade` command instead with the updated config. We don't want to recommend users modify resources managed by helm.

Reply:
changed